### ISSUE TITLE
export RS prefixes information

### ISSUE THREAD

**Issue by thomas-mangin** [pierky/invalidroutesreporter#1]
Dear Pier Carlo,

One of my bugbear with IX RS usage reports is that they tend to report the number of prefixes available at the exchange over time. I argue that this metric is useless (at least alone) as a /8 and /24 both count as a single entry. 
As it is common for network to announce more-specific route to influence network routing, I would rather see reports of the numbers of unique smallest routable entries ( /24 for IPv4 - not sure from the top of my head for IPv6) present at the RS. Obviously, de-duplication of overlapping IP would need to occur.

Your mission @pierky, should you choose/decide to accept it, would be to implement a way to record currently announced unique /24 present at the RS to allow people to graph them over time.
Ideally the number reported should be per ASN.

A technical way would be to implement a new webserver exposing the data for a program like Prometheus to collect. Having used a similar approach (Prometheus export) for another project, I think it is a good way to go. Should you have any question, please contact me offline and I will share some code with you.

As always, should you or any of your team be caught or killed, the internet community will disavow any knowledge of your actions. This issue will self-destruct in ten seconds. Good luck @pierky

**Comment by pierky** [comment]
It has not been the easiest mission I faced but I hope I accomplished it in a right way for the sake of a better Internet: https://github.com/pierky/usres_monitor

Currently there is not an use case yet but I plan to add an example based on ExaBGP: a script that receives announces and withdraws and that periodically exports info to $destination.

Since this topic is not strictly related to InvalidRoutesReporter I close this issue here and kindly ask the huge mass of people who want to comment to open new issues on the new repository.

P.S.: @thomas-mangin thanks for the inspiration!
